### DISCUSSION TITLE
AI Search

### DISCUSSION THREAD

**Discussion by ibrahim-gabol-19** [ibrahim-gabol-19/comp0016-systems-engineering-group-11-2024#30] [General]
@MohamadHabbal 

Here is the trip advisor version of ai search. 
- I think the idea of text and cards together is good.
- We could do them horizontally.
- We could embed them into each text bullet point (a lot of work!)

What do you think?

![image](https://github.com/user-attachments/assets/90cc559c-fa57-419c-b919-61483f51770c)

**Comment by MohamadHabbal** [comment]
Yeah that could work. Horizontally would definitely look better. Maybe a maximum of 2 or 3 cards next to each other.
